content/en/security/application_security/troubleshooting.md

```diff
@@ -564,28 +564,41 @@ See the Application Security [set up documentation][11] to validate you you are
 
 Ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable (`DD_TRACE_TELEMETRY_ENABLED` for NodeJS) is set to `true`, or the corresponding system property for your language is enabled. For example in Java: `-Ddd.instrumentation.telemetry.enabled=true`
 
-## Disabling Threat Detection and Protection
+## Disabling threat detection and protection
 
 To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration, and restart your service.
 
 If no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
 * If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false`, and restart your service.
-* If ASM was activated in one click: go to [ASM service configuration][15], select **Services with Threat Management Capabilities**, click on your service, and then click **Deactivate**.
-* To disable ASM on your services in bulk, click the check box in the list header and then under **Bulk Actions** select **Deactivate Threat Management detection on (number of) services**.
-
-## Disabling Software Composition Analysis (SCA)
+* If ASM was activated in one click, do the following: 
+  1. Go to [ASM service configuration][15].
```

Comment:
Might be good to rephrase go to ASM > Catalog > Services since we moved it in the main nav under the catalog section.